### DISCUSSION TITLE
Can we expect an update to Kotlin 1.9.0?

### DISCUSSION THREAD

**Discussion by herrbert74** [Foso/Ktorfit#386] [Q&A] [answered]
I'm fairly new to KMP and I'm trying to convert my Android app. However, I ran into a config cache issue, that is fixed on Kotlin 1.9.0. However, it seems that a few libraries are in lockstep with Kotlin versions, and Ktorfit is the only one that hasn't updated yet, so it's a blocker for me. Can we expect an update, and timely updates to Kotlin versions in general?

Do you need any help with updates? I seen that you use renovate/debendabot. I have a lot of Android experience, however zero with dependabot/renovate.

Sorry if I come across as impatient, I'm really grateful for the library.

I also tried to build the library and examples at one point. That might have failed due to my limited experience with KMP, but it seems to me that the sandbox module has a few nonexistent classes, thus failing the build. The AndroidOnlyExample seems to be missing configurations and the app gradle file completely.
Also, targeting Java 8 through the toolchain seems to fail on M1 Macs, due to the missing Java 8 release from AdoptOpenJDK. Changing it to Java 11 fixed it. For building the examples with AGP 8.0 you will also need to change the compilation to Java 17.

**Comment by Foso** [comment, marked as answer]
Hi, yes, I'm planning to release it at the weekend. The last two KSP versions had bugs which broke Ktorfit and I couldn't update it. But yesterday a KSP Hotfix was released, and that fixed the bugs and I already updated the project yesterday

**Reply by herrbert74** [reply]
Got it, thanks!
